src/soroban_rpc.ts

```diff
@@ -39,6 +39,17 @@ export namespace SorobanRpc {
     latestLedger?: number;
   }
 
+  /* Response for jsonrpc method `getNetwork`
+   */
+  export interface GetNetworkResponse {
+    friendbotUrl?: string;
+    latestLedgerHash: string;
+    latestLedgerSeq: number;
+    latestLedgerTime: string;
+    passphrase: string;
+    protocolVersion: string;
+  }
```

Comment:
Can we reconcile this with the return values in [the doc](https://docs.google.com/document/d/1TZUDgo_3zPz7TiPMMHVW_mtogjLyPL0plvzGMsxSz6A/edit#heading=h.3zrwisrq3rg3)? It's hard to know where the source of truth is when reviewing fields/types/etc.

Reply:
oh duh. Yeah. This response is wrong, sorry!